File: app/src/main/java/nolambda/stiker/motionviews/serialized/TextEntitySavedState.java
package nolambda.stiker.motionviews.serialized;

import nolambda.stiker.motionviews.viewmodel.Layer;

public class TextEntitySavedState extends BaseEntitySavedState {
    public TextEntitySavedState(Layer layer,
                                float[] matrixValues,
                                float holyScale,
                                int canvasWidth,
                                int canvasHeight,
                                PaintData borderPaintData) {
        super(layer, matrixValues, holyScale, canvasWidth, canvasHeight, borderPaintData);
    }
}

File: app/src/main/java/nolambda/stiker/motionviews/serialized/TextEntitySavedState.kt
package nolambda.stiker.motionviews.serialized

import nolambda.stiker.motionviews.viewmodel.Layer

class TextEntitySavedState(
    layer: Layer,
    matrixValues: FloatArray,
    holyScale: Float,
    canvasWidth: Int,
    canvasHeight: Int,
    borderPaintData: PaintData
) : BaseEntitySavedState(layer, matrixValues, holyScale, canvasWidth, canvasHeight, borderPaintData)
